apps/storefront/src/pages/ShoppingListDetails/components/ReAddToCart.tsx

```diff
@@ -258,6 +264,36 @@ export default function ReAddToCart(props: ShoppingProductsProps) {
     }
   };
 
+  const handleReAddAddToCartBackend = async () => {
+    setLoading(true);
+
+    try {
+      const lineItems = addLineItems(products);
+      const res = await callCart(lineItems);
```

Comment:
[Suggestion]

if we can rename the function `callCart` in: https://github.com/bigcommerce/b2b-buyer-portal/blob/feat/b2b-3321/apps/storefront/src/utils/cartUtils.ts#L121-L129

to provide context on what it does actually that'd be great - since it's used all over the place.

I think a better name would be `createOrUpdateExistingCart`